packages/ffe-tabs/less/tab.less

```
@@ -9,63 +9,77 @@
 // Styleguide ffe-tabs.tab
 
 .ffe-tab {
-    background-color: @ffe-grey-cloud;
-    border: solid 2px @ffe-grey-silver;
-    border-bottom: none;
-    color: @ffe-grey-charcoal;
-    cursor: pointer;
-    display: block;
+    display: flex;
+    background-color: @ffe-farge-hvit;
+    color: @ffe-farge-fjell;
+    padding: @ffe-spacing-xs @ffe-spacing-md;
+    border-radius: 40px;
+    white-space: nowrap;
+    border: 2px solid transparent;
+    margin: @ffe-spacing-2xs;
     font-family: 'SpareBank1-regular', 'MuseoSans-500', arial, sans-serif;
-    font-size: 1rem;
-    outline: none;
-    overflow: hidden;
-    padding: @ffe-spacing-xs @ffe-spacing-lg @ffe-spacing-xs;
-    text-align: center;
-    text-decoration: none;
+    align-self: stretch;
+    justify-content: center;
     transition: all @ffe-transition-duration @ffe-ease;
-    vertical-align: middle;
+    cursor: pointer;
+
     .native & {
         @media (prefers-color-scheme: dark) {
-            background-color: @ffe-grey-charcoal-darkmode;
-            border-color: @ffe-grey-silver-darkmode;
-            color: @ffe-grey-cloud-darkmode;
+            color: @ffe-farge-vann-30;
+            background-color: @ffe-farge-svart;
         }
     }
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: @ffe-blue-royal;
-        background: @ffe-white;
+    &:focus {
+        box-shadow: 0 0 0 3px @ffe-farge-hvit, 0 0 0 5px @ffe-farge-vann;
+        outline: 0;
+
         .native & {
             @media (prefers-color-scheme: dark) {
-                background: @ffe-black-darkmode;
-                color: @ffe-blue-azure-darkmode;
+                box-shadow: 0 0 0 3px @ffe-farge-svart,
+                    0 0 0 5px @ffe-farge-hvit;
             }
         }
     }
 
-    &:focus,
-    &:active {
-        box-shadow: 0 0 0 2px @ffe-blue-focus;
-        outline: 0;
+    &:focus:not(:focus-visible) {
+        outline: none;
+        box-shadow: none;
     }
 
-    &--selected {
-        background-color: @ffe-white;
-        color: @ffe-blue-royal;
+    &:hover {
+        color: @ffe-farge-hvit;
+        background-color: @ffe-farge-vann;
+
         .native & {
             @media (prefers-color-scheme: dark) {
-                background: @ffe-black-darkmode;
-                color: @ffe-blue-azure-darkmode;
+                background-color: @ffe-farge-vann-70;
+                color: @ffe-farge-svart;
             }
         }
     }
 
-    &--condensed {
-        font-size: 0.875rem;
-        padding-bottom: @ffe-spacing-2xs;
-        padding-top: @ffe-spacing-2xs;
+    &--selected,
+    &:active {
+        background-color: @ffe-farge-fjell;
+        color: @ffe-farge-hvit;
+
+        .native & {
```

Tror ikke `&` trengs her :blush: 
```suggestion
        .native {
```

---

Hvis jeg fjerner den funker ikke css for darkmode :thinking:

---

Da så :smile:

---

`&` plassert bakerst sånn som dette indikerer at du gjentar klassen du er nøstet ned til, så den er viktig å la stå for å ende opp med riktig selector. :)

Kompilert resultat i dette tilfellet er 
```
.native .ffe-tab--selected,
.native .ffe-tab:active {
```

---

Ja, ikke sant. Tenkte helt feil. Uten ville den vel kun stylet elementer med class "native" innenfor klassen den er nøstet i. :see_no_evil: